packages/smart-contracts/test/common.ts

```diff
@@ -49,3 +49,45 @@ export const tryAndWait = async (f: any) => {
     }
   }
 };
+
+export const getTokenFromUniswapAndApproveProxyTransfer = async (
```

Comment:
What button does this correspond to?

Reply:
We dont nees to add this feature. Its only for testing